docs/CONTRIBUTING.md

```diff
@@ -23,9 +24,11 @@ you'd like to address unless the proposed fix is minor.
 
 In your Pull Requests (PR), link to the issue that the PR solves.
 
-Please ensure that the base of your PR is the **master** branch. (Note: The default
-GitHub branch is the latest release branch, though you should point all of your changes to be merged into
-master).
+Please ensure that the base of your PR is the **main** branch.
+
+### Commits
+
+We prefer a clean commit history. This means you should squash all commits before asking for review (cleanup, squash, force-push). If you need help with this, feel free to leave a comment in your PR and we'll guide you.
```

Comment:
`squash all commits` == PRs should only have one commit? Or does that just mean to squash fixups and fixup-type commits?

Reply:
Squash fixups and fixup-type commits - let me reword to clarify! Good point.

Reply:
fixed!